File: app/src/main/java/com/smith/ufc/character/CharacterRepository.java
package com.smith.ufc.character;

import com.smith.ufc.data.base.Repository;
import com.smith.ufc.data.models.MarvelCharacter;
import com.smith.ufc.data.service.MarvelCharacterDataSource;

/**
 * Created by Charlton on 10/31/17.
 */

public interface CharacterRepository extends Repository<MarvelCharacter, CharacterSpecification>, MarvelCharacterDataSource {}

File: app/src/main/java/com/smith/ufc/data/repositories/character/CharacterRepository.kt
package com.smith.ufc.data.repositories.character

import com.smith.ufc.data.base.BaseRepository
import com.smith.ufc.data.models.MarvelCharacter
import com.smith.ufc.data.service.MarvelCharacterDataSource

/**
 * Created by Charlton on 10/31/17.
 */

interface CharacterRepository : BaseRepository<MarvelCharacter, CharacterSpecification>, MarvelCharacterDataSource
